ga4gh/backend.py

```diff
@@ -17,6 +17,167 @@
 import ga4gh.datamodel.variants as variants
 
 
+def _parsePageToken(pageToken, numValues):
+    """
+    Parses the specified pageToken and returns a list of the specified
+    number of values. Page tokens are assumed to consist of a fixed
+    number of integers seperated by colons. If the page token does
+    not conform to this specification, raise a InvalidPageToken
+    exception.
+    """
+    tokens = pageToken.split(":")
+    # TODO define exceptions.InvalidPageToken and raise here.
+    if len(tokens) != numValues:
+        raise Exception("Invalid number of values in page token")
+    # TODO catch a ValueError here when bad integers are passed and
+    # convert this into the appropriate InvalidPageToken exception.
+    values = map(int, tokens)
+    return values
+
+
+def _getVariantSet(request, variantSetIdMap):
```

Comment:
Might as well inline this into `VariantsIntervalIterator._getContainer` for symmetry with `ReadsIntervalIterator`.